assets/css/style.css

```
@@ -1,23 +1,45 @@
+.oss-section {
+  --color-background: var(--oss-bg);
+  --color-surface: var(--oss-surface);
+  --color-text: var(--oss-text);
+  --color-muted: var(--oss-muted);
+  --color-accent: var(--oss-accent);
+  --color-border: var(--oss-border);
+  --color-surface-secondary: rgba(139, 148, 158, 0.12);
+  --color-surface-tertiary: rgba(139, 148, 158, 0.2);
+  --color-header-gradient-start: #161b22;
+  --color-header-gradient-end: #0d1117;
+  font-family: system-ui, -apple-system, sans-serif;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)